chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/CertifiedProductManager.java

```diff
@@ -1079,6 +1081,7 @@ private void logCertifiedProductCreateActivity(Long listingId) throws JsonProces
     // listings collection is not evicted here because it's pre-fetched and
     // handled in a listener
     // no other caches have ACB data so we do not need to clear all
```

Comment:
Are these comments still relevant?

Reply:
I believe this comment is still relevant.  It is referring to "listings collection" and the ListingStore does not have any interaction with the collections.